test/members_page_test.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+require_relative './test_helper'
+require_relative '../lib/members_page.rb'
+
+describe 'MembersPage' do
+  around { |test| VCR.use_cassette('Members Page', &test) }
+
+  describe '#to_h' do
+    subject do
+      url = 'http://sitl.diputados.gob.mx/LXIII_leg/listado_diputados_gpnp.php?tipot=TOTAL'
+      MembersPage.new(response: Scraped::Request.new(url: url).response).to_h
+    end
+
+    it 'should have a :party_groupings field' do
+      subject.to_h[:party_groupings].wont_be_nil
+    end
+  end
+
+  describe 'party groupings' do
+    subject do
+      url = 'http://sitl.diputados.gob.mx/LXIII_leg/listado_diputados_gpnp.php?tipot=TOTAL'
```

Comment:
There's to much duplication of this URL (and indeed of instantiating the MembersPage from it)— this could usefully be factored out somewhere.